swagger/swagger.yaml

```diff
@@ -49,8 +49,8 @@ paths:
           default: false
       - name: max_scenarios
         in: query
-        description: "the maximum number of scenarios requested, supported only in\
-          \ `test` sessions"
+        description: "the maximum number of scenarios requested, not supported in\
```

Comment:
The previous description shows that this was only used for 'test' sessions (Not eval or ta1) but they were removed. By saying that it's not supported in 'eval' sessions does this mean it is supported in ta1? Or does is this just no longer supported

Reply:
Correct-- it is supported in TA1 (`soartech` & `adept`) sessions, but not `eval` sessions.  Basically I removed `test`, but made sure that `max_scenarios` works in `soartech` & `adept`.